src/BuildingRegistry.Projections.Legacy/BuildingSyndication/BuildingSyndicationProjections.cs

```diff
@@ -1068,6 +1068,30 @@ await context
                 }, ct);
             });
 
+            When<Envelope<BuildingUnitRemovalWasCorrected>>(async (context, message, ct) =>
+            {
+                await context.CreateNewBuildingSyndicationItem(
+                    message.Message.BuildingPersistentLocalId,
+                    message,
+                    x =>
+                    {
+                        var buildingUnitSyndicationItem = new BuildingUnitSyndicationItemV2
+                        {
+                            PersistentLocalId = message.Message.BuildingUnitPersistentLocalId,
+                            Status = BuildingRegistry.Building.BuildingUnitStatus.Parse(message.Message.BuildingUnitStatus),
+                            Function = BuildingRegistry.Building.BuildingUnitFunction.Parse(message.Message.Function),
+                            Position = message.Position,
+                            PointPosition = message.Message.ExtendedWkbGeometry.ToByteArray(),
+                            PositionMethod = BuildingRegistry.Building.BuildingUnitPositionGeometryMethod.Parse(message.Message.GeometryMethod),
+                            Version = message.Message.Provenance.Timestamp,
+                            Addresses = new Collection<BuildingUnitAddressSyndicationItemV2>()
```

Comment:
Isn't it possible that this has an addressid? We already have migrated units with addressid's

Reply:
I'll take this up.